firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -154,6 +154,13 @@ public void setUserId(@NonNull String identifier) {
     core.setUserId(identifier);
   }
 
+  /**
+   * @returns the unique identifier for the current user or empty string if it was not specified.
+   */
+  public @NonNull String getUserId() {
+    return core.getUserId() == null ? "" : core.getUserId();
```

Comment:
Note to reviewer: this should be `@Nullable`, but the reason why I added the `@NonNull` annotation is kotlin requires the getter to have the same annotation as the setter's parameter (hence the empty string being returned).